editor.planx.uk/src/@planx/components/AddressInput/model.ts

```diff
@@ -3,7 +3,7 @@ import { object, string } from "yup";
 
 import { MoreInformation, parseMoreInformation } from "../shared";
 
-export type UserData = {
+export type PersonAddress = {
```

Comment:
nit: this component is also used to collect organisation addresses - maybe `ApplicantAddress` or `UserAddress` is a bit more generalized and fits better with existing language?

Reply:
Yeah this is a good point 👍 On the XML side of things I'm trying to make it distinct from a site address and generating an address for either the agent or applicant so `ApplicantAddress` or `UserAddress` doesn't quite work either.

Could it be as simple as just `Address` and `SiteAddress`?

Reply:
Yep happy with `Address` & `SiteAddress` !